libs/ledger-live-common/src/families/cosmos/js-synchronisation.ts

```diff
@@ -37,9 +39,17 @@ const txToOps = (info: AccountShapeInfo, accountId: string, txs: CosmosTx[]): Co
   for (const tx of txs) {
     let fees = new BigNumber(0);
 
-    tx.tx.auth_info.fee.amount.forEach(elem => {
-      if (elem.denom === unitCode) fees = fees.plus(elem.amount);
-    });
+    if (tx.tx && tx.tx.auth_info && tx.tx.auth_info.fee && tx.tx.auth_info.fee.amount) {
+      fees = tx.tx.auth_info.fee.amount.reduce(
```

Comment:
Just for readability
```suggestion
    const amounts = tx?.tx?.auth_info?.fee?.amount;
    const fees = !amounts ? new BigNumber(0) : amounts.reduce(
```